requirements.txt

```diff
@@ -1,4 +1,9 @@
+model-archiver
+multi-model-server==1.1.1
 numpy
+psutil>=5.6.7  # sagemaker-containers requires psutil 5.6.7
```

Comment:
Do we need this line? It doesn't looks like we were specifying the version before, and this isn't locking the version to 5.6.7 either.